telemetry/main.schema.json

```diff
@@ -846,13 +846,16 @@
           "minimum": 0
         },
         {
-          "type": "string"
+          "type": "string",
+          "pattern": "^[a-zA-Z][a-zA-Z0-9_.]*[a-zA-Z0-9]$"
```

Comment:
@trink I have no idea how to do that and what is the syntax to accomplish what you suggest...
If you could guide me I will do it and push again!